tests/functional/builtins/codegen/test_keccak256.py

```diff
@@ -18,10 +18,10 @@ def bar() -> bytes32:
 
     c = get_contract_with_gas_estimation(hash_code)
     for inp in (b"", b"cow", b"s" * 31, b"\xff" * 32, b"\n" * 33, b"g" * 64, b"h" * 65):
-        assert "0x" + c.foo(inp).hex() == keccak(inp).hex()
+        assert c.foo(inp).hex() == keccak(inp).hex()
```

Comment:
should we remove the calls to `.hex()` now? i guess they make error messages a little easier to read (when the assertion fails)

Reply:
let's remove it, the error would still be readable